packages/core/src/errors/transaction.ts

```diff
@@ -65,3 +65,16 @@ export class WaitForTransactionReceiptTimeoutError extends BaseError {
     )
   }
 }
+
+export type TransactionsFetchErrorType = TransactionsFetchError & {
+  name: 'TransactionsFetchError'
+}
+
+export class TransactionsFetchError extends BaseError {
```

Comment:
Maybe we could name this `TransactionsNotFoundError` to be similar to the existing `TransactionNotFoundError` but just use a plural form.